backend/index.js

```diff
@@ -0,0 +1,78 @@
+/*---------------Funciones--------------------*/
+function actualizarContador() {
+    const checkboxes = document.querySelectorAll('input[type="checkbox"]');
+    let contador = 0
+    checkboxes.forEach(checkbox => {
+        if (checkbox.checked) {
+            contador++;
+        }
+    });
+    document.getElementById('contador').innerText = contador
+}
+/*------------------Crear botones----------------*/
+function guardar() {
```

Comment:
La funcion es muy larga y podria ser dificil de debuggear
Ademas  no es necesario   recorrer la tabla con un for. 
Podriamos aprovechar y separar la logica Por ejemplo a partir de la 19 podriamos hacer algo parecido  a esto
` const table = document.getElementById('tbodyTask');
    const tr = document.createElement('tr');

    const checkboxColumna = crearCheckboxColumna();
    const tareaColumna = crearTareaColumna(tarea);
    const eliminarColumna = crearEliminarColumna();

    tr.appendChild(checkboxColumna);
    tr.appendChild(tareaColumna);
    tr.appendChild(eliminarColumna);

    tablaTareas.appendChild(tr);`